src/main/java/Main.java

```diff
@@ -0,0 +1,21 @@
+import java.util.ArrayList;
+import java.util.NoSuchElementException;
+
+public class Main {
+
+    public static void main(String[] args) {
+        try {
+            ArrayList<Integer> list = new ArrayList<>();
+            list.add(7);
+            list.add(3);
+            list.add(9);
+            list.add(21);
+            list.add(18);
+            System.out.println("Sum of list " + Functional.sum(list));
+            System.out.println("Max in list " + Functional.max(list));
+        } catch (NoSuchElementException n_s_e) {
```

Comment:
And again, because of `NoSuchElementException` is a subclass of `RuntimeException` it is possible to ignore this exception.